pry.gemspec

```diff
@@ -20,5 +21,13 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'coderay',       '~> 1.1.0'
   s.add_dependency 'method_source', '~> 0.9.0'
+
+  if Pry::Platform.mri_19? # 1.9 compatible.
```

Comment:
@banister is this logic better placed inside the Gemfile? when do we plan to drop 1.9 support?

Reply:
outdated by af929e95bae5cd3f78b7a712804ae826c85c2f43

Reply:
changed a little bit and then squashed into 96d3d87ecc4886a9f77fe48f6a14a9171a58ecf5